import-automation/progress-dashboard-rest/test/test_base_database.py

```diff
@@ -15,48 +15,54 @@
 """
 Tests for base_database.py.
 
-The Datastore emulator must be running for these tests to run.
-In a different terminal, run
-'gcloud beta emulators datastore start --no-store-on-disk'.
-In the terminal where the tests will be run, run
-'$(gcloud beta emulators datastore env-init)' and proceed to run the tests.
+The Datastore emulator must be installed. See
+https://cloud.google.com/datastore/docs/tools/datastore-emulator#before_you_begin
+for requirements.
 
-See https://cloud.google.com/datastore/docs/tools/datastore-emulator.
+TODO(intrepiditee): decide later whether test filenames start with
+test_ or end with _test or use a pattern that supports both.
 """
 
 import unittest
+from unittest import mock
 
-from google.auth import credentials
 from google.cloud import datastore
 
-from app import utils
 from app.service import base_database
+from test import utils
 
 
 class BaseDatabaseTest(unittest.TestCase):
     """Tests for BaseDatabase."""
 
+    @classmethod
+    def setUpClass(cls):
```

Comment:
Do we need to do this at the class level xor the per-test level?  If it is possible for a test failure to goof up the emulator's state so other tests fail, we need it at the per-test level.  Thanks.